code-for-canada/project-thundercat
# [Submit Test Flow] Real Content Insert

# Description

I added the real content in the submit test popup box in both official languages.

## Type of change

Please delete options that are not relevant.

- Code cleanliness or refactor

## Screenshot

**English Version:**

![image](https://user-images.githubusercontent.com/23021242/54353911-9fcc1180-462b-11e9-8229-d516171ee7cd.png)

**French Version:**

![image](https://user-images.githubusercontent.com/23021242/54353919-a8bce300-462b-11e9-831a-9918048c40fb.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB sample test.
2.  Complete the pre-test process.
3.  Click Submit test and make sure that the content is right (in both official languages).

Screenshot of unit tests run passing:

![image](https://user-images.githubusercontent.com/23021242/54354046-06e9c600-462c-11e9-80dd-5c38a13d4a4e.png)

# Checklist

Applicable for all code changes.
